qiskit_braket_provider/providers/braket_job.py

```diff
@@ -119,6 +120,42 @@ def shots(self) -> int:
     def submit(self):
         return
 
+    def queue_position(self) -> QuantumTaskQueueInfo:
```

Comment:
queue_position and queue_info are two separate methods for IBM Jobs [[Ref](https://qiskit.org/documentation/stable/0.19/_modules/qiskit/providers/ibmq/job/ibmqjob.html#IBMQJob.queue_position)] Is it possible for us to align to make it easier for qiskit customers to easily switch between IBM and Braket?